packages/beacon-node/src/chain/bls/multithread/jobItem.ts

```diff
@@ -70,15 +69,15 @@ export function jobItemWorkReq(job: JobQueueItem, format: PointFormat, metrics:
       // and not a problem in the near future
       // this is monitored on v1.11.0 https://github.com/ChainSafe/lodestar/pull/5912#issuecomment-1700320307
       const timer = metrics?.blsThreadPool.signatureDeserializationMainThreadDuration.startTimer();
```

Comment:
this metric is misleading now, need to change it to a new name and description, and update grafana bls dashboard